src/viam/rpc/dial.py

```diff
@@ -66,6 +66,10 @@ class DialOptions:
     if detected, even with credentials present. This is generally
     unsafe to use, but can be requested."""
 
+    timeout: float = 20
+    """Number of seconds before the dial connection times out
+    Set to 20sec to match _defaultOfferDeadline in goutils/rpc/wrtc_call_queue.go"""
```

Comment:
This shouldn't be in the docstring -- it can be a comment, but it doesn't need to be publicly available in the docs